Assets/MixedRealityToolkit.SDK/StandardAssets/Shaders/HandTriangles.shader

```diff
@@ -611,9 +611,6 @@ SubShader {
         UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(vxIn[0]);
         //huxEye = _WorldSpaceCameraPos;
         //workaround for Unity's auto updater in 5.6
```

Comment:
Looks like the code was removed below, so should the comment be removed as well?

Reply:
I left the code in sense this shader was generated from a tool, and it may be easier to resolve changes if we take an update from that tool.